src/main/java/gui/SnappingPane.java

```diff
@@ -28,8 +28,9 @@
     public SnappingPane(int row, int column, double width, double height) {
         this.row = row;
         this.column = column;
-        this.setWidth(width);
-        this.setHeight(height);
+        // don't enforce width, let grid handle this - Mark
+        //this.setWidth(width);
+        //this.setHeight(height);
```

Comment:
Could you remove these lines of commented code when they are not used, commented code is never pleasant